desktop-ui/desktop-ui.cpp

```diff
@@ -144,6 +163,17 @@ auto nall::main(Arguments arguments) -> void {
     if(file::exists(argument)) program.startGameLoad.append(argument);
   }
 
+#if defined(PLATFORM_WINDOWS)	
+ /* Force and Change Settings By Commandline */
```

Comment:
nit: we don't tend to use block comments for single line comments, use // here

Reply:
Comment Removed.